.github/workflows/build-container.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
+    if: github.repository == 'opensciencegrid/docker-software-base'
```

Comment:
I am guessing this is right how you have it - but what sets this `github.repository` ?  (Do you have a reference doc where you saw it?)

Reply:
Recommended here: https://github.community/t/stop-github-actions-running-on-a-fork/17965/2

[GHA contexts](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contexts) can make use of the [github object](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context), where `github.repository` contains the owner and repo name where the action is running.

Reply:
Great, thanks for the info/reference